src/desktop/cio.desktop

```diff
@@ -13,6 +13,7 @@ X-SuSE-YaST-AutoInst=
 X-SuSE-YaST-Geometry=
 X-SuSE-YaST-SortKey=
 X-SuSE-YaST-AutoInstResource=
+X-SuSE-YaST-Keywords=blacklist,io,channels,system
```

Comment:
why system? and maybe something like s390? as it is s390 specific module only? Maybe also `cio_ignore` as it is name of script that is used. and maybe also `zconf` as it is tool for s390 that can do it low level.

Reply:
Good points. I have added.